src/components/Avatar/Avatar.css

```diff
@@ -1,62 +0,0 @@
-/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
-@import 'src/css/variables/borders.css';
-@import 'src/css/variables/colors.css';
-
-.y-avatar {
-  position: relative;
-}
-
-.y-avatar__size-xLarge {
-  height: 72px;
-  width: 72px;
-}
-.y-avatar__size-large {
-  height: 48px;
-  width: 48px;
-}
-.y-avatar__size-medium {
-  height: 40px;
-  width: 40px;
-}
-.y-avatar__size-small {
-  height: 32px;
-  width: 32px;
-}
-.y-avatar__size-xSmall {
-  height: 24px;
-  width: 24px;
-}
```

Comment:
Seems these rules have no impact on our diffs.  @swese44 do you remember why we have them?

Reply:
Fabric upgrades have broken these dimensions a few times with margins and overflows, i feel we should continue to have these here

Reply:
Ok.  I've added them back.  Would those regressions be tough to catch with visual diffs?

Reply:
they'll be caught, but we'll have to re-add these css rules to fix them

Reply:
i went through the file history and i think it was the nested image component which has broken in the past, and that css has been removed after the PersonaCoin refactor. feel free to remove, sorry for the churn :(

Reply:
No problem. I'll remove the height and width from the root again.